raster/src/main/scala/geotrellis/raster/render/ColorRamp.scala

```diff
@@ -125,8 +125,21 @@ object ColorRamp {
     * @param n       Length of list to return
     */
   def spread(colors: Vector[Int], n: Int): Vector[Int] = {
+    /* The requested spread matches the original */
     if (colors.length == n) return colors
 
+    /* 2016 May 20 @ 14:21
```

Comment:
Eh, my personal opinion is that it adds noise, and it's not clear why it's there, and there's not a standard for the codebase around doing it, so I'd rather it not be there. @echeipesh do you have any opinions on the matter?